helm/charts/determined/values.yaml

```diff
@@ -128,8 +127,8 @@ maxSlotsPerPod: 8
   # cpu: 7
 
 # Memory and CPU requirements for the master instance. Should be adjusted for scale.
-masterCpuRequest: 2
-masterMemRequest: 8Gi
+masterCpuRequest: 8
```

Comment:
these values need to match your scheam.json property names so in the schema you have `resources.memory` where it might need to be called `masterCpuRequest`. etc.

Reply:
that is if they are to be configurable.

Reply:
This is for the master deployment. for the training jobs, I fetch it from values.schema.json as seen in values.yaml in the `{{.Values.*}`